### ISSUE TITLE
Enable single instance mode (optional)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#41]
```
As a user I would like to see support for a single instance of pino. I have 
e dock-like interface and when I minimize pino it stays in the tray. This is 
all fine. But when I click on the icon in my dock (or gnome-menu) pino 
creates another instance. But I would like pino to active the existing 
instance.
```

Original issue reported on code.google.com by `mattijs....@gmail.com` on 13 Jan 2010 at 7:55

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 13 Jan 2010 at 11:02
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Hi, i have made a small patch to implement this feature, it will add libunique 
as a 
dependency, please try it out and tell me what do you think.
It also include the spanish translation for the new items in the preferences 
dialog.

Best regards.
```

Original comment by `paulo.ri...@gmail.com` on 30 Jan 2010 at 3:30

Attachments:
- [multiple_instances.patch](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-41/comment-2/multiple_instances.patch)

**Comment by GoogleCodeExporter** [comment]
```
Closed by 15b38975ea10 revision.

@paulo.richards, can you give me your email? I want to add you to the authors 
list.
```

Original comment by `tro...@gmail.com` on 20 Feb 2010 at 2:28
- Changed state: **Fixed**
